src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/Program.cs

```diff
@@ -25,7 +25,9 @@
 #if (IndividualLocalAuth)
 using Company.WebApplication1.Data;
 #endif
+#if (OrganizationalAuth || IndividualB2CAuth || IndividualLocalAuth || MultiOrgAuth || GenerateGraph)
```

Comment:
This fixes this:

![image](https://user-images.githubusercontent.com/54385/126709363-85225a0e-04d5-45b9-bab5-c325b49fe06d.png)

I don't like how this needs to be kept in sync with the reset of the `using` `#if`'s, but I don't see a better way.